source/MRMesh/MRGcodeProcessor.h

```diff
@@ -28,6 +31,7 @@ class MRMESH_CLASS GcodeProcessor
     struct MoveAction
     {
         BaseAction3f action;
+        std::vector<Vector3f> toolDirection; // tool direction for each point action.path
```

Comment:
what is this for?

Reply:
removed. will using in next PR for show tool direction in GcodeVisualizer